roles/kubernetes-apps/external_provisioner/local_path_provisioner/templates/local-path-storage-cm.yml.j2

```diff
@@ -11,6 +11,57 @@ data:
                 {
                         "node":"DEFAULT_PATH_FOR_NON_LISTED_NODES",
                         "paths":["{{ local_path_provisioner_claim_root }}"]
+                },
+                {
+                        "node":"yasker-lp-dev1",
```

Comment:
what is `yasker-lp-dev1` ?

Reply:
Got it from here https://github.com/rancher/local-path-provisioner/blob/v0.0.19/deploy/example-config.yaml

But should not be needed in this kubespray config. I will remove this and the other dev3 one.